lib/shoulda/matchers/active_model/allow_value_matcher.rb

```diff
@@ -589,7 +589,7 @@ def inspected_values_to_set
           if values_to_set.size > 1
             values_to_set.map(&:inspect).to_sentence(
               two_words_connector: " or ",
-              last_word_connector: ", or"
+              last_word_connector: ", or "
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.<br>Put a comma after the last parameter of a multiline method call.